ubuntu/code-server.sh

```diff
@@ -30,5 +46,16 @@ if [ $USER = ubuntu ]; then
 fi
 EOF
 
-# Install extensions
-#sudo -iu ubuntu code-server code-server --install-extension ...
+# Install extensions, if any
+if [[ $code_server_extensions && ${code_server_extensions-_} ]]; then
```

Comment:
```suggestion
if [[ ${code_server_extensions} && ${code_server_extensions-_} ]]; then
```

Reply:
On pourrait pas se passer du `if` en itérant simplement sur une liste éventuellement vide ?